metals/src/main/scala/scala/meta/internal/metals/NewFilesProvider.scala

```diff
@@ -181,4 +188,7 @@ class NewFilesProvider(
         |}
         |""".stripMargin
 
+  private def caseClassTemplate(name: String): String =
+    s"case class $name()"
```

Comment:
Add final?

Reply:
I thought about it, but it seemed a bit too opinionated? On the other hand, I never use final on case classes because I’m too lazy to type it so this may be what makes me start using it 😅

Reply:
I feel like the custom wizard is more merited if it’s more than just adding “case”

Reply:
Hey, it also adds different parenthesis! :D 

Anyway, added `final`, I think it makes sense